webapp/src/pages/DepositPage/DepositPage.tsx

```diff
@@ -329,30 +329,24 @@ const HeroSection: React.FC<{
     const Logo = getAssetLogo(asset);
 
     switch (asset) {
-      case "WETH":
-        return <Logo />;
       case "WBTC":
-        return <Logo height="190%" style={{ marginTop: 40 }} />;
```

Comment:
@fqlx , I've fixed this manual margin displacement here by simply resizing the logo to twice the allocated size